### PR TITLE
fix(theme): remove a fixed content height

### LINKED ISSUE #836: When ngFor used within accordion body accordion does not resize appropriately.
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description
When ngFor is used within an accordion body, the body does not resize appropriately when toggled. The body only seems to resize to static content. 

**Steps to reproduce:**

**Related code:**
```
<nb-accordion>
      <nb-accordion-item>
                <nb-accordion-item-body>
                          <div *ngFor="let q of questions">{{ q }}</div>
                 </nb-accordion-item-body>
      </nb-accordion-item>
</nb-accordion>
```

**Angular, Nebular**
```
Angular: 6.1.7
Nebular: 2.0.0-rc.10

### PR DESCRIPTION
**Pull request by lexzhukov**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

Closes #836

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nebular",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (nnixaa) [inline package-lock.json]: let's keep this change separated from this PR

File: src/framework/theme/components/accordion/accordion-item-body.component.ts [hunk 2]
```diff
@@ -4,15 +4,7 @@
...
-import {
-  Component,
-  ChangeDetectionStrategy,
-  Host,
-  ElementRef,
-  ChangeDetectorRef,
-  OnInit,
-  OnDestroy,
-} from '@angular/core';
+import { Component, ChangeDetectionStrategy, Host, ChangeDetectorRef, OnInit, OnDestroy } from '@angular/core';
```
Comment (nnixaa) [inline src/framework/theme/components/accordion/accordion-item-body.component.ts]: is this necessary?

Comment (lexzhukov) [reply src/framework/theme/components/accordion/accordion-item-body.component.ts]: it's not important, but more readable now.

File: package-lock.json (hunk 1 above)
Comment (lexzhukov) [reply package-lock.json]: Ok, I'll remove.

File: src/framework/theme/components/accordion/accordion-item-body.component.ts (hunk 2 above)
Comment (nnixaa) [reply src/framework/theme/components/accordion/accordion-item-body.component.ts]: :) it's debatable, I suggest we not mix code style changes into the particular issue fixes

Comment (lexzhukov) [reply src/framework/theme/components/accordion/accordion-item-body.component.ts]: Fixed.

**Pull request merged**